src/Imdb/Title.php

```diff
@@ -1262,12 +1262,16 @@ public function sound()
     public function mpaa()
     {
         if (empty($this->mpaas)) {
-            $this->getPage("ParentalGuide");
-            if (preg_match_all("|/search/title\?certificates=.*?>\s*(.*?):(.*?)<|", $this->page["ParentalGuide"],
-                $matches)) {
-                $cc = count($matches[0]);
-                for ($i = 0; $i < $cc; ++$i) {
-                    $this->mpaas[$matches[1][$i]] = $matches[2][$i];
+            $xpath = $this->getXpathPage("ParentalGuide");
+            if (empty($xpath)) {
+                return array();
+            }
+            $cells = $xpath->query("//section[@id=\"certificates\"]//li[@class=\"ipl-inline-list__item\"]");
+            foreach ($cells as $cell) {
+                if ($cell->getElementsByTagName('a')->item(0) !== null) {
```

Comment:
After seeing #237 maybe we should do the same here.

```
if ($a = $cell->getElementsByTagName('a')->item(0)) {
    $mpaa = explode(':', $a->nodevalue);
    $this->mpaas[trim($mpaa[0])] = $mpaa[1];
}
```

PS. What does `$te` stand for?

Reply:
$te is the output of explode and contains country and certificate value.
But your change is fine, except that there is a fault in your example above..
nodeValue versus nodevalue makes a lot of sence haha

Reply:
I can see what you assign it to, but not why it where named `$te`, what are `te` short for? e.g. `$cc` would have been `$country_certificate`.

Reply:
Yeah you are right again, i have to learn to use more inventive var names.
$mpaa is for now acceptable as you suggested?

Reply:
Naming convention are one of the hardest part of coding! Code looks good, will need to run it locally before merging, can't do it from where I'm typing as of right now.

Reply:
Agree sometimes it is very hard to come up with a meaningful name, laziness slips in and i use some kind of characters that don't mean anything but works..